addon/components/o-s-s/dialog.ts

```diff
@@ -46,4 +50,8 @@ export default class OSSDialog extends BaseModal<OSSDialogArgs> {
   onInit(elem: HTMLElement): void {
     this.initialize(elem, false);
   }
+
+  private isSafeString(data: any): boolean {
+    return typeof data === 'object' && !isEmpty(data.string);
+  }
```

Comment:
According to the `ember-intl` module, you can maybe use `import type { SafeString } from '@ember/template';` for the typing

Reply:
@nathalieArnoux 🙏 As Julien write, ember-template already has type & validator, it could be great to use them instead
https://api.emberjs.com/ember/5.11/functions/@ember%2Ftemplate/isHTMLSafe